src/app/assets/containers/asset-detail/asset-detail.component.html

```diff
@@ -52,10 +52,20 @@ <h2>{{theAsset.name}}</h2>
     <div class="asset-details-card mat-elevation-z8" *ngFor="let theDefendant of (defendants$ | async)">
       <app-view-defendant
         [defendant]="theDefendant"
+        *ngIf="getPropertyState(theDefendant.id) === 'view'"
         [identifiers]="identifiers$ | async"
         [isDeleting]="(isDefendantDeleting$(theDefendant.id) | async)"
         (defendantDeleted)="onDefendantDeleted($event)"
+        (defendantEdit)="setPropertyStateEdit(theDefendant.id)"
       ></app-view-defendant>
+      <app-edit-defendant
+        [defendant]="theDefendant"
+        [identifiers]="identifiers$ | async"
+        *ngIf="getPropertyState(theDefendant.id) === 'edit'"
```

Comment:
Can we create a method in the component in order to not hardcode strings, especially in the template, please?